packages/projects-docs/pages/learn/sandboxes/editors.md

```diff
@@ -13,7 +13,7 @@ import Video from '../../../../../shared-components/Video'
 
 CodeSandbox offers two web editors and an iOS editor. A Sandbox is either a Browser Sandbox or a Cloud Sandbox (read more [here](/learn/sandboxes/overview)). All Sandboxes are available on the iOS app. 
 
-CodeSandbox also supports VS Code for [Repositories](/learn/repositories/overview)
+CodeSandbox supports VS Code for Cloud Sandboxes and [Repositories](/learn/repositories/overview).
```

Comment:
should we add a link to Cloud Sandbox info as well?

Reply:
Definitely! Thanks for the suggestion.